src/lib/components/popup/AddCredential.svelte

```diff
@@ -62,11 +71,11 @@
     addCredentialPayload.name = name;
     addCredentialPayload.description = description;
     addCredentialPayload.userFields = userFields;
-    console.log(addCredentialPayload);
     await addCredential(addCredentialPayload);
     showFolderList = false;
+    //Need to dispatch add credetial
+    closeEventDispatcher();
     await browser.windows.remove(windowId);
```

Comment:
handle manual here?

Reply:
You mean, Dispatch from there itself?
Without calling another function?
'